source/Public/Utils/Get-FabricLongRunningOperationResult.ps1

```diff
@@ -33,14 +33,9 @@ This command fetches the result of the operation with the specified operationId.
 
     try {
         # Step 2: Make the API request
-        $response = Invoke-RestMethod `
-            -Headers $FabricConfig.FabricHeaders `
+        $response = Invoke-FabricRestMethod `
             -Uri $apiEndpointUrl `
```

Comment:
also should this have the full api url?

Reply:
Not part of this PR, but I can change and use `$FabricConfig.BaseUrl` to avoid a hard-coded base URL.

Reply:
No, it's smart enough to accept full url or part url :)